FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -81,7 +85,58 @@ struct IssueDone: View {
             AnalyticsManager.shared.trackPageview(path: "/issue/\(issue.slug)/done/")
             
             store.dispatch(action: .FetchStats(issue.id))
+         
+            // will prompt for a rating after hitting done 5 times
+            RatingPromptCounter.increment {
+                guard let currentScene = UIApplication.shared.connectedScenes.first as? UIWindowScene else {
+                    return
+                }
+                
+                SKStoreReviewController.requestReview(in: currentScene)
+            }
+            
+            // unlikely to occur at the same time as the rating prompt counter
+            checkForNotifications()
+        }.alert(R.string.localizable.notificationTitle(), isPresented: $showNotificationAlert) {
+            Button {
+                // we don't really care which issue they were on when they subbed, just that it was a done page
+                AnalyticsManager.shared.trackEvent(name: "push-subscribe", path: "/issue/x/done/")
+                OneSignal.promptForPushNotifications(userResponse: { (success) in
```

Comment:
```suggestion
                OneSignal.promptForPushNotifications(userResponse: { _ in })
```

Reply:
unless you meant to handle the response

Reply:
We actually only track this if the call succeeds in reminders, so I'll do the same here